scc/provider/resource_subaccount.go

```diff
@@ -78,6 +77,18 @@ __Further documentation:__
 				Computed:            true,
 				Optional:            true,
 			},
+			"connected": schema.BoolAttribute{
```

Comment:
I think the documentation should state that this is steering whether the subaccount should be connected (true) or disconnected (false) in an analogy to the API description